core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -70,7 +70,7 @@ public function normalizeMetadata(StoreDbafsMetadataEvent $event): void
             $event->set('importantPartHeight', $importantPart->getHeight());
         }
 
-        if (\is_array($data = $extraMetadata['metadata'])) {
+        if (isset($extraMetadata['metadata']) && \is_array($data = $extraMetadata['metadata'])) {
```

Comment:
```suggestion
        if (\is_array($data = $extraMetadata['metadata'] ?? null)) {
```

Reply:
I thought about this too, but I find my version more readable (and it does not unnecessarily call `is_array` after an "internal" isset()-call).

Reply:
Don't agree, tbh. I find the other version more readable and it's probably even faster for the common path (`metadata` exists in `tl_files`). :smile: